tensorflow/python/ops/map_fn.py

```diff
@@ -477,6 +479,8 @@ def compute(i, tas):
       elems_value_flat = _elems_value_batchable_to_flat(elems_value_batchable,
                                                         elems_flat_signature)
       elems_value = elems_unflatten(elems_value_flat)
+      autographed_fn = autograph.tf_convert(fn, autograph_ctx.control_status_ctx())
+      result_value = autographed_fn(elems_value)
       result_value = fn(elems_value)
```

Comment:
Remove duplicate line - we must not call the original `fn` again.

Reply:
Done